packages/emitter-framework/src/typescript/components/enum-declaration.tsx

```diff
@@ -61,7 +57,7 @@ export function EnumMember(props: EnumMemberProps) {
     <ts.EnumMember
       name={props.type.name}
       jsValue={props.type.value ?? props.type.name}
-      refkey={ay.refkey(props.refkey ?? props.type)}
+      refkey={props.refkey ?? efRefkey(props.type)}
```

Comment:
This default is also suspicious...

Reply:
I removed the defaults and addressed the rest of the feedback. Assuming CI passes I'll hit you for another review. Thanks!